src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * The maximum bitrate available for this guild
+   * @type {number}
+   * @readonly
+   */
+  get maximumBitrate() {
+    if (this.features.includes('VIP_REGIONS')) {
+      return 384000;
+    } 
+    
+    switch (PremiumTiers[this.premiumTier]) {
+      case 0:
```

Comment:
```suggestion
      default:
      case 0:
```
seeing as they are the same this is valid

Reply:
Not even sure why you would suggest invalid code.

Reply:
True. This is not working. Switch back to the previous default.

Reply:
bit of a derp the logic behind it yes, did edit though to suggest a much more logical approach

Reply:
ah i see, the code was invalid because i thought it would be implied to remove the already existing default case

but of course as i said removing the 0 case is a smarter approach

Reply:
Yea, that's was i was going to commit.